doc/user/content/get-started/quickstart.md

```diff
@@ -307,4 +309,7 @@ DROP TABLE fraud_accounts;
 
 [//]: # "TODO(morsapaes) Extend to suggest third party tools. dbt, Census and Metabase could all fit here to do interesting things as a follow-up."
 
-To get started with your own data, [upgrade your playground to a trial account](https://materialize.com/trial/?utm_campaign=General&utm_source=documentation).
+- To get started with your own data, click the `Connect data` button in the
+  Console.
+
+- To get a guided tour, click the `Talk to us` button in the Console.
```

Comment:
```suggestion
To get started ingesting your own data from an external system like Kafka, MySQL
or PostgreSQL, check the documentation for [sources](/sql/create-source/), and
navigate to **Data** > **Sources** > **New source** in the [Materialize Console](https://console.materialize.com/)
to create your first source.
```

Reply:
This original CTAs make sense for users that are following the embedded quickstart in the console, but not for users who are following the quickstart in the documentation (which _might_ include users running the Docker image, which has no notion of a UI). Left a suggestion to point users to the source creation UI in the console, but also to the `CREATE SOURCE` landing page (which should really be an overview page for `Ingest data`, but we don't have one just yet).